### PR TITLE
Relax pytest version range

### PR DESCRIPTION
**Pull request by john-kurkowski**
~~What do you think about supporting Python 2.7?  I'm personally stuck on Python 2.7 a while longer. There wasn't much Python 3-specific code in this repo.~~ **EDIT:** reverted.

~~Also,~~ I'm using pytest 3.0.x personally. Seems to work here. I didn't see this repo using any >3.3 specifics.

### CODE REVIEW

**Comment by henry0312** [comment]
Thank you for your contribution. but I'm sorry I will never support Python 2.7.x.
And, If you send a PR to support more pytest versions, I'll merge.

**Comment by john-kurkowski** [comment]
I've reverted the Python 2.7 commit. That way, the decision and diff are there for posterity. I've retitled the PR to be just about relaxing the required pytest version.

File: setup.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
     py_modules=['pytest_codestyle'],
     python_requires='>=3.6,<4',
     install_requires=[
-        'pytest>=3.3,<3.4',
+        'pytest>=3.0,<3.4',
```
Comment (henry0312) [inline setup.py]: Please also change `<3.4` to `<4` :)
I assume that pytest follow [Semantic Versioning](https://semver.org/).

**Comment by henry0312** [comment]
I'm sorry for rate response.
I'm back from new year vacation.

File: setup.py (hunk 1 above)
Comment (john-kurkowski) [reply setup.py]: Done

**Comment by henry0312** [comment]
Thank you!

**Pull request merged**